compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -265,6 +266,16 @@ def start_endpoint(
                 )
                 Endpoint.pidfile_cleanup(endpoint_dir)
 
+        if endpoint_config.endpoint_setup:
+            try:
+                subprocess.check_call(endpoint_config.endpoint_setup, shell=True)
+            except Exception:
+                log.exception(
+                    "unable to execute endpoint_setup command: "
+                    + endpoint_config.endpoint_setup
+                )
```

Comment:
From the pedantic department, we _did_ execute it, but the command returned a non-zero exit status.  It _failed_.  Perhaps something like:
```python
f"endpoint_setup script failed: {...}"
```
(This is bike-shed, so pick your own poison.)